src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -29,6 +29,132 @@ class OrchestratorCli(orchestrator.OrchestratorClientMixin, MgrModule):
         {'name': 'orchestrator'}
     ]
 
+    def __init__(self, *args, **kwargs):
+        super(OrchestratorCli, self).__init__(*args, **kwargs)
+        self.ident = set()  # type: Set[str]
+        self.fault = set()  # type: Set[str]
+        self._load()
+        self._refresh_health()
+
+    def _load(self):
+        active = self.get_store('active_devices')
+        if active:
+            decoded = json.loads(active)
+            self.ident = set(decoded.get('ident', []))
+            self.fault = set(decoded.get('fault', []))
+        self.log.debug('ident %s, fault %s' % (self.ident, self.fault))
+
+    def _save(self):
+        encoded = json.dumps({
```

Comment:
```
encoded = json.dumps({
            'ident': list(self.ident),
            'fault': list(self.fault),
            })
```
looks a little bit more compact and intuitive IMHO.
You're doing this in ``_command_ls`` yourself :-)

Reply:
fixed